src/metadata-args/types/ColumnMode.ts

```diff
@@ -7,6 +7,7 @@
 export type ColumnMode =
     | "regular"
     | "virtual"
+    | "calculated"
```

Comment:
let's go with "virtual-property" instead of "calculated" for now.

Reply:
Sounds good. I will update this in a few hours.

Reply:
This has been updated